src/Microsoft.ML.Data/Transforms/Hashing.cs

```diff
@@ -834,6 +1083,66 @@ private void AddMetaKeyValues(int i, DataViewSchema.Annotations.Builder builder)
             }
 
             protected override Delegate MakeGetter(DataViewRow input, int iinfo, Func<int, bool> activeOutput, out Action disposer) => _parent.GetGetterCore(input, iinfo, out disposer);
+
+            private bool SaveAsOnnxCore(OnnxContext ctx, int iinfo, string srcVariable, string dstVariable)
+            {
+                string opType;
+                OnnxNode murmurNode;
+
+                opType = "MurmurHash3";
+                if (_types[iinfo].RawType == typeof(KeyDataViewType))
```

Comment:
It seems like the output for this transformer is always uint32 for the tests in your branch. Do you have a test where this code get's called?

Reply:
No, I don't have.